resolwe/flow/models/entity.py

```diff
@@ -30,6 +30,20 @@ def duplicate(self, contributor, inherit_collection_membership=False):
             for entity in self
         ]
 
+    @transaction.atomic
+    def move_to_collection(self, source_collection, destination_collection):
```

Comment:
Move the logic in to the model, so you can also move a single object. You can keep a wrapper here, to use it on the queryset.